client/app/achievements/achievements.less

```diff
@@ -1,3 +1,7 @@
 ng-invald {
   color: red
 }
+
+.event-description > img {
+  width: 100%;
```

Comment:
Tested with a markdown image and it works as expected. Going to go with this for now
![screenshot 2016-09-27 at 10 43 58 am](https://cloud.githubusercontent.com/assets/2281120/18878622/545f636e-849f-11e6-8bba-2f6b9dd8d54c.png)